src/components/10-atoms/button/index.react.d.ts

```diff
@@ -20,7 +20,7 @@ export interface AXAButtonProps {
   size?: Size;
   motionOff?: boolean;
   disabled?: boolean;
-  onClick?: () => void;
+  onClick?: (event: React.MouseEvent<HTMLElement>) => void;
```

Comment:
I trust you on this being correct ;-)

Reply:
We have exact this syntax at other components.